text/rfcs/README.md

```diff
@@ -0,0 +1 @@
+# RFCs
```

Comment:
What is the intention for this page?

Reply:
mdbook requires everything in the summary.md file to be a link. No raw text. So, readme file to link to.